File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Temperature.java
package com.riders.thelab.data.remote.dto.weather;

import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Temperature {
    @Json(name = "day")
    private double day;
    @Json(name = "min")
    private double min;
    @Json(name = "max")
    private double max;
    @Json(name = "night")
    private double night;
    @Json(name = "eve")
    private double evening;
    @Json(name = "morn")
    private double morning;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Temperature.kt
package com.riders.thelab.data.remote.dto.weather

import kotlinx.serialization.SerialName
import java.io.Serializable

@kotlinx.serialization.Serializable
data class Temperature(
    @SerialName("day")
    val day: Double = 0.0,

    @SerialName("min")
    val min: Double = 0.0,

    @SerialName("max")
    val max: Double = 0.0,

    @SerialName("night")
    val night: Double = 0.0,

    @SerialName("eve")
    val evening: Double = 0.0,

    @SerialName("morn")
    val morning: Double = 0.0
) : Serializable
